website/next-i18next.config.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
       "vi",
       "zh",
       "tr",
+      "id-ID",
```

Comment:
Other locales are using just the language code.  How essential is the country code here?

Reply:
Thank you for the feedback! After a second thought, you are right. We don't need the country ID here. I updated the PR to only use language code.

Reply:
Looks like this array entry needs to be updated as well to be `id`.